android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/NewPostActivity.java

```diff
@@ -196,18 +196,26 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (requestCode == TC_PICK_IMAGE) {
             if (resultCode == Activity.RESULT_OK) {
                 final boolean isCamera;
-                if (data.getData() == null) {
-                    isCamera = true;
-                } else {
-                    isCamera = MediaStore.ACTION_IMAGE_CAPTURE.equals(data.getAction());
-                }
-                if (!isCamera) {
-                    mFileUri = data.getData();
+                if(data!=null) {
```

Comment:
Please add space between keyword and opening parenthesis, and spaces around "!=".